.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,5 @@
+This pull request includes
```

Comment:
I’m not sure if I would know how to answer this…

What about something like “What does this PR do?”